tests/tests_app/utilities/packaging/projects/Dockerfile.cpu

```diff
@@ -1 +1 @@
-FROM pytorchlightning/pytorch_lightning:base-cpu-py3.7-torch1.8
+FROM pytorchlightning/pytorch_lightning:base-cuda-py3.9-torch1.13-cuda11.7.1
```

Comment:
AFAIK we don't have a `base-cpu` image.

Reply:
no, it was not worse for the building...